dart/lib/src/transport/http_transport.dart

```diff
@@ -1,5 +1,6 @@
 import 'dart:async';
 import 'dart:convert';
+import 'dart:io';
```

Comment:
This breaks Dart for Web/Flutter for Web as dart:io is not available on web.
The gzip encoding should probably go into `encode.dart` with a corresponding no op in `noop_encode.dart`

Reply:
Hmm, ok. This would also mean that we can't use `GZipCodec` on web, and therefor would have to encode the whole payload again, or do you know of any other way to do this?

Reply:
Your welcome. On web it isn't encoded at all, I think.

Reply:
Fixed, thank you. 🙇